app/github_public_repo/README.md

```diff
@@ -0,0 +1,20 @@
+
+# SARS-CoV-2 Serosurveillance Data by Serotracker
+
+SeroTracker synthesizes findings from thousands of COVID-19 seroprevalence studies worldwide, providing a data platform and interactive dashboard for SARS-CoV-2 serosurveillance. This dataset represents our collection of serosurveillance studies
+
+### Download
+Our complete SARS-Cov-2 dataset is available as a CSV programmatically with the following command:
+- `curl https://raw.githubusercontent.com/serotracker/sars-cov-2-data/main/test.csv`.
+
+Otherwise, you can also download our dataset directly from our [Airtable view]([https://airtable.com/shraXWPJ9Yu7ybowM/tbljN2mhRVfSlZv2d?backgroundColor=blue&viewControls=on](https://airtable.com/shraXWPJ9Yu7ybowM/tbljN2mhRVfSlZv2d?backgroundColor=blue&viewControls=on)).
+
+The CSV file follows the format of 1 row per serosurveillance study.
+
+  
+### Additional Information
+Please see our [Data Dictionary]([https://docs.google.com/spreadsheets/d/1KQbp5T9Cq_HnNpmBTWY1iKs6Etu1-qJcnhdJ5eyw7N8/edit#gid=0](https://docs.google.com/spreadsheets/d/1KQbp5T9Cq_HnNpmBTWY1iKs6Etu1-qJcnhdJ5eyw7N8/edit#gid=0)) for explanations of our variables, data types, and descriptions as well as insight into how our data is collected by our research team.
```

Comment:
@annoyingpr please confirm that our data dictionary has all the columns that were are showing in the csv.

Reply:
seems like we're missing two columns: `Last Modified Time` and `Data Quality Status`. i've messaged Christian about including them!

Reply:
Does the data dictionary have the `database_names` for each variable?

Reply:
right now the names are the same, but are not reflected in snake case (e.g. right now it is "Prevalence Estimate Name" vs "prevalence_estimate_name")? @rahularoradfs should this be updated? i can pass on to @123chrisc who's doing the data dict updates

Reply:
As discussed in the basecamp thread in the Data Infrastructure group, we want our data dictionary to contain all the snake case names of the human readable names. For example, we want our data dict to include both "Prevalence Estimate Name" and "prevalence_estimate_name" so it is easy for users to map these. @annoyingpr can you:

1) make sure all the fields in this airtable table are in the data dictionary. If there are missing fields, please add them to the data dict https://airtable.com/appFfYIupTrVQ0HJx/tbliKC06AgJ1EAKLa/viwoxvB5i6kITkaiF?blocks=hide

2) make sure all the snake casing of fields is added to our data dictionary. you can use the 'full_airtable_fields' dict in this file to create this mapping correctly: https://github.com/serotracker/iit-backend/blob/master/app/utils/airtable_fields_config.py